src/Server/Commands/PublicKeyCommand.php

```diff
@@ -29,15 +29,16 @@ class PublicKeyCommand extends Command
      */
     public function handle()
     {
-        $url = config('auth.providers.northstar.key');
+        // Get Northstar URL from config.
+        $url = config('auth.providers.northstar.url');
         if (! $url) {
-            $key = config('services.northstar.key');
+            $url = config('services.northstar.url');
```

Comment:
@DFurnes is the difference between this and line 33 just if it using OAuth or not?

Reply:
I moved the "suggested" place for these two things to be configured into `config/auth.php`, since it makes more sense to handle all the server stuff in the same place. These conditionals are fallbacks so that the old config locations would work too (but they don't work very well since I made a typo on what variable to set here!!)

Reply:
Same deal with the line added in the other file –– `auth.providers.northstar.key` is the new location for the same thing that was previously set at `services.northstar.key`. Whenever we make a major version bump of this library, we can remove support for the old way of doing things.

Reply:
ah makes sense to me now - thanks for explaining!!